setup.py

```diff
@@ -38,7 +38,8 @@ def get_version(filename):
 
 
 TESTS_REQUIRE = [
-    'pytest'
+    'pytest',
+    'wheel'
```

Comment:
?

Reply:
Needed to build the package, as `wheel` is not installed by default on the GH runner. Alternatively, I could add `python3 -m pip install wheel` in the cylc/release-actions build action https://github.com/cylc/release-actions/blob/master/build-python-package/action.yml ?